src/core_modules/capture-core/components/WidgetTwoEventWorkspace/OverflowMenu/Modal/UnlinkAndDeleteModal.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import i18n from '@dhis2/d2-i18n';
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+} from '@dhis2/ui';
+import log from 'loglevel';
+import { useDataEngine, useAlert } from '@dhis2/app-runtime';
+import { useMutation, useQueryClient } from 'react-query';
+import { ReactQueryAppNamespace } from 'capture-core/utils/reactQueryHelpers';
+import type { Props } from './UnlinkAndDeleteModal.types';
+
+export const UnlinkAndDeleteModal = ({
+    setOpenModal,
+    eventId,
+    originEventId,
+}: Props) => {
+    const dataEngine = useDataEngine();
+    const queryClient = useQueryClient();
+    const { show: showErrorAlert } = useAlert(
+        i18n.t('An error occurred while unlinking and deleting the event.'),
+        { critical: true },
+    );
+
+    const deleteEvent = async () => {
+        const mutation = {
+            resource: 'tracker?async=false&importStrategy=DELETE',
+            type: 'create',
+            data: { events: [{ event: eventId }] },
+        };
+
+        return dataEngine.mutate(mutation);
+    };
+
+    const mutation = useMutation(deleteEvent, {
+        onSuccess: () => {
+            queryClient.invalidateQueries([
+                ReactQueryAppNamespace,
+                'linkedEventByOriginEvent',
+                originEventId,
+            ]);
+            setOpenModal(false);
+        },
+        onError: (error) => {
+            showErrorAlert();
+            log.error(
+                `Failed to unlink and delete event with ID: ${eventId}`,
+                error,
+            );
+        },
+    });
+
+    return (
+        <Modal dataTest="event-unlink-and-delete-modal" position="middle">
```

Comment:
The two models have different positions. I think most of the existing models are "on top" of the page. Can you remove `position="middle"` to align them? Thanks!

<img width="2092" alt="Screenshot 2024-11-06 at 09 22 42" src="https://github.com/user-attachments/assets/97bb82bb-fa20-4218-aa03-c70bbb53c1ff">